packages/caido/PKGBUILD

```
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=caido
+pkgname=('caido-cli' 'caido-desktop')
+pkgver=0.52.0
+pkgrel=1
+pkgdesc='A lightweight web security auditing toolkit.'
+url='https://caido.io/'
+license=('custom:unknown')
```

```suggestion
license=('custom:private')
```

It would be also better to scrap https://caido.io/fr/terms and copy it to `install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"`